com.unity.template-universal/Packages/manifest.json

```diff
@@ -6,9 +6,7 @@
     "com.unity.analytics": "3.6.11",
```

Comment:
Same as in the package.json file, I don't think the first 4 packages should be there. Is this intended?

Reply:
I have simply copy/paste existing manifest package list, I have no idea what is really required :)